docs/11.1/guides/getting-started/demo.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Demo
+metaTitle: Demo - Guide - Handsontable Documentation
+permalink: /11.1/demo
+canonicalUrl: /demo
+tags:
+  - demo
+  - hello world
+---
+
+# Demo
+
+[[toc]]
+
+Play around with Handsontable's source code, in your favorite framework.
+
+---
+
+<BigExample preview="/examples/next/docs/js/demo/">
+  <BigExampleSource 
+  label="JavaScript"
+  icon="js"
+  target="/examples/next/docs/js/demo/"></BigExampleSource>
+  <BigExampleSource 
+  label="TypeScript"
+  icon="ts"
+  target="/examples/next/docs/ts/demo/"></BigExampleSource>
+  <BigExampleSource 
+  label="Angular"    
+  icon="angular"
+  target="/examples/next/docs/angular/demo/"></BigExampleSource>
+  <BigExampleSource 
+  label="React"
+  icon="react"
+  target="/examples/next/docs/react/demo/"></BigExampleSource>
+  <BigExampleSource 
+  label="Vue"
+  icon="vue"
+  target="/examples/next/docs/vue/demo/"></BigExampleSource>
+</BigExample>
```

Comment:
There was some unnecessary padding at the bottom of the example, but I pushed a commit that fixes it:

![image](https://user-images.githubusercontent.com/566463/152237290-2f130005-6bf6-4c99-ad09-c7f103d826d2.png)